Text/index.jsx

```diff
@@ -20,7 +21,7 @@ Text.propTypes = {
   children: PropTypes.node,
   size: PropTypes.oneOf(['large', 'small', 'extra-small']),
   weight: PropTypes.oneOf(['bold', 'thin']),
-  color: PropTypes.oneOf(['white']),
+  color: PropTypes.oneOf(['white', 'gray']),
```

Comment:
I like that we're only adding colors that we need here 😄  My hope is that we continue to use a base set of colors that we use for everything, so we have a simple color palette that we can use for anything from Text, Buttons, Borders etc. @stevemdixon wondering how that sounds to you there since I know you've thought about the colors?